app/src/main/java/com/keylesspalace/tusky/AccountActivity.kt

```diff
@@ -247,13 +241,11 @@ class AccountActivity : BottomSheetActivity(), ActionButtonActivity, HasSupportF
             override fun onOffsetChanged(appBarLayout: AppBarLayout, verticalOffset: Int) {
 
                 @AttrRes val attribute = if (titleVisibleHeight + verticalOffset < 0) {
-                    accountToolbar.setTitleTextColor(textColorPrimary)
```

Comment:
Will this color ever be set now? Does it happen automatically

Reply:
Thats the default color, it happens automatically